docs/user/reference/environment-variables.rst

```diff
@@ -16,13 +16,14 @@ All :doc:`build processes </builds>` have the following environment variables au
 .. envvar:: READTHEDOCS_LANGUAGE
 
     The locale name, or the identifier for the locale, for the project being built.
-    This value comes from the project's configured language.
+    This value comes from the project's configured language,
+    normalized to be lowercase and use a dash as a separator instead of an underscore.
```

Comment:
I thought we mentioned this somewhere else, but doesn't seem like it. So, +1 on having a note about it.

```suggestion
    This value comes from the project's configured language,
    the language code is in lowercase and use a dash as a separator.
```

Reply:
How about this: 

```suggestion
    This value comes from the project's configured language code,
    which is in lowercase and use a dash as a separator.
```

Alternatively, a "and" before "the language code" to connect the two parts of this sentence.

Reply:
Sounds good!